releasenotes/notes/fix-bit-copy-4b2f7349683f616a.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Fixed an issue with copying circuits with new-style :class:`.Clbit`\ s and
+    :class:`.Qubit`\ s (bits without registers) where references to these bits
+    from the containing circuit could be broken, causing issues with
+    serialization and circuit visualization.
```

Comment:
Can we add a unit test to enforce that this remains fixed?  It's a pretty subtle bug, and would be easy to accidentally re-introduce.

Reply:
Done in 0943608c3cd0f75f83b47d64191b4f7460b51611.